resources/views/robots-txt.phtml

```diff
@@ -27,7 +27,27 @@ Disallow: <?= $base_path ?>/admin
 Disallow: <?= $base_path ?>/manager
 Disallow: <?= $base_path ?>/editor
 Disallow: <?= $base_path ?>/account
-Crawl-delay: 10
+Crawl-delay: 1
+
+Disallow: <?= $base_path ?>/login   
+Disallow: <?= $base_path ?>/module 
+
+Disallow: /*/*-list
```

Comment:
You cannot use `*` here.   See http://www.robotstxt.org/robotstxt.html

> Note also that globbing and regular expression are not supported in either the User-agent or Disallow lines. The '*' in the User-agent field is a special value meaning "any robot". Specifically, you cannot have lines like "User-agent: *bot*", "Disallow: /tmp/*" or "Disallow: *.gif".

Reply:
I made this changes in robots.txt 3 month ago in my installation and it's works.

Google recommends using Disallow in robots.txt instead of Not Follow,

I'll open a new threat for Google index in open discussion later this week